### PR TITLE
Release Candidate: ofrak 2.1.0

### PR DESCRIPTION
**Pull request by EdwardLarson**
**One sentence summary of this PR (This should go in the CHANGELOG!)**
Release ofrak 2.1.0, ofrak_patch_maker 3.0.0, and ofrak_type 2.1.0

**Link to Related Issue(s)**

**Please describe the changes in your request.**

**Anyone you think should look at this, specifically?**

### CODE REVIEW

File: ofrak_core/ofrak/core/ubi.py [hunk 1]
```diff
@@ -36,6 +39,24 @@
...
+    def __init__(self):
+        super().__init__(
+            "python-lzo",
+            "https://github.com/jd-boyd/python-lzo",
+            install_check_arg="",
+        )
+
+    def is_tool_installed(self) -> bool:
+        try:
+            return True
```
Comment (whyitfor) [inline ofrak_core/ofrak/core/ubi.py:52]:
```suggestion
            import lzo
            return True
```

File: ofrak_type/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ All notable changes to `ofrak-type` will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased](https://github.com/redballoonsecurity/ofrak/tree/master)
+
+## [2.1.0](https://github.com/redballoonsecurity/ofrak/releases/tag/ofrak-type-v2.1.0) - 2023-01-20
```
Comment (whyitfor) [inline ofrak_type/CHANGELOG.md:8]: One could make an argument that PatchMaker should be 3.0.0, given the removal of `ToolchainVersion` and updated constructor for `PatchMaker`. `ofrak` setup.py would then need to be updated.

**Review by whyitfor — COMMENTED** [review]
See a couple comments

File: ofrak_core/ofrak/core/ubi.py [hunk 3]
```diff
@@ -36,6 +39,25 @@
...
+    def __init__(self):
+        super().__init__(
+            "python-lzo",
+            "https://github.com/jd-boyd/python-lzo",
+            install_check_arg="",
+        )
+
+    def is_tool_installed(self) -> bool:
+        try:
+            import lzo
```
Comment (whyitfor) [inline ofrak_core/ofrak/core/ubi.py:52]:
```suggestion
            import lzo  # type: ignore
```

File: ofrak_core/CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,8 @@ All notable changes to `ofrak` will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased](https://github.com/redballoonsecurity/ofrak/tree/master)
+
+## [2.1.0](https://github.com/redballoonsecurity/ofrak/releases/tag/ofrak-v2.1.0) - 2023-01-20
```
[the remark this answers is not in the document]
Comment (EdwardLarson) [reply ofrak_core/CHANGELOG.md]: Done

**Review by whyitfor — APPROVED** [review]

**Pull request merged**